diff --git a/build/webcrypto.es.js b/build/webcrypto.es.js
index f5c3d08f352fb127b57b1d59161dfe5fbf69e4ae..59e80563c96f274f07263738266e9c3248564498 100644
--- a/build/webcrypto.es.js
+++ b/build/webcrypto.es.js
@@ -6,7 +6,7 @@ import * as core from 'webcrypto-core';
 import { BufferSourceConverter as BufferSourceConverter$1 } from 'webcrypto-core';
 export { CryptoKey } from 'webcrypto-core';
 import { Buffer } from 'buffer';
-import * as crypto from 'crypto';
+import crypto from 'crypto';
 import crypto__default from 'crypto';
 import * as process from 'process';
 import { __decorate } from 'tslib';
diff --git a/package.json b/package.json
index 5107e80a6acaa0560dc20c2fad9d925b7ef1c4a2..460844b76e3e3209b22d07745cb5282083be895a 100644
--- a/package.json
+++ b/package.json
@@ -14,6 +14,7 @@
   ],
   "main": "build/webcrypto.js",
   "module": "build/webcrypto.es.js",
+  "react-native": "build/webcrypto.es.js",
   "types": "index.d.ts",
   "scripts": {
     "test": "mocha",
