diff --git a/node_modules/@decentralized-identity/ion-tools/node_modules/@noble/ed25519/lib/esm/index.js b/node_modules/@decentralized-identity/ion-tools/node_modules/@noble/ed25519/lib/esm/index.js
index 70147e8..fa5e392 100644
--- a/node_modules/@decentralized-identity/ion-tools/node_modules/@noble/ed25519/lib/esm/index.js
+++ b/node_modules/@decentralized-identity/ion-tools/node_modules/@noble/ed25519/lib/esm/index.js
@@ -756,7 +756,7 @@ export const curve25519 = {
 };
 const crypto = {
     node: nodeCrypto,
-    web: typeof self === 'object' && 'crypto' in self ? self.crypto : undefined,
+    web: undefined
 };
 export const utils = {
     TORSION_SUBGROUP: [
diff --git a/node_modules/@decentralized-identity/ion-tools/node_modules/@noble/ed25519/lib/index.js b/node_modules/@decentralized-identity/ion-tools/node_modules/@noble/ed25519/lib/index.js
index 811d1c7..eee32d9 100644
--- a/node_modules/@decentralized-identity/ion-tools/node_modules/@noble/ed25519/lib/index.js
+++ b/node_modules/@decentralized-identity/ion-tools/node_modules/@noble/ed25519/lib/index.js
@@ -769,7 +769,7 @@ exports.curve25519 = {
 };
 const crypto = {
     node: crypto_1.default,
-    web: typeof self === 'object' && 'crypto' in self ? self.crypto : undefined,
+    web: undefined
 };
 exports.utils = {
     TORSION_SUBGROUP: [
