diff --git a/node_modules/@decentralized-identity/ion-sdk/node_modules/@noble/secp256k1/lib/esm/index.js b/node_modules/@decentralized-identity/ion-sdk/node_modules/@noble/secp256k1/lib/esm/index.js
index ff76464..cccf0c5 100644
--- a/node_modules/@decentralized-identity/ion-sdk/node_modules/@noble/secp256k1/lib/esm/index.js
+++ b/node_modules/@decentralized-identity/ion-sdk/node_modules/@noble/secp256k1/lib/esm/index.js
@@ -1046,7 +1046,7 @@ export const schnorr = {
 Point.BASE._setWindowSize(8);
 const crypto = {
     node: nodeCrypto,
-    web: typeof self === 'object' && 'crypto' in self ? self.crypto : undefined,
+    web: undefined
 };
 const TAGS = {
     challenge: 'BIP0340/challenge',
diff --git a/node_modules/@decentralized-identity/ion-sdk/node_modules/@noble/secp256k1/lib/index.js b/node_modules/@decentralized-identity/ion-sdk/node_modules/@noble/secp256k1/lib/index.js
index e207e74..fcfcab2 100644
--- a/node_modules/@decentralized-identity/ion-sdk/node_modules/@noble/secp256k1/lib/index.js
+++ b/node_modules/@decentralized-identity/ion-sdk/node_modules/@noble/secp256k1/lib/index.js
@@ -1056,7 +1056,7 @@ exports.schnorr = {
 Point.BASE._setWindowSize(8);
 const crypto = {
     node: nodeCrypto,
-    web: typeof self === 'object' && 'crypto' in self ? self.crypto : undefined,
+    web: undefined
 };
 const TAGS = {
     challenge: 'BIP0340/challenge',
