diff --git a/node_modules/@noble/secp256k1/lib/esm/index.js b/node_modules/@noble/secp256k1/lib/esm/index.js
index f2484fe..403e3f3 100644
--- a/node_modules/@noble/secp256k1/lib/esm/index.js
+++ b/node_modules/@noble/secp256k1/lib/esm/index.js
@@ -1085,9 +1085,10 @@ export const schnorr = {
     verifySync: schnorrVerifySync,
 };
 Point.BASE._setWindowSize(8);
+
 const crypto = {
     node: nodeCrypto,
-    web: typeof self === 'object' && 'crypto' in self ? self.crypto : undefined,
+    web: undefined,
 };
 const TAGS = {
     challenge: 'BIP0340/challenge',
diff --git a/node_modules/@noble/secp256k1/lib/index.js b/node_modules/@noble/secp256k1/lib/index.js
index 33a0843..6874ed7 100644
--- a/node_modules/@noble/secp256k1/lib/index.js
+++ b/node_modules/@noble/secp256k1/lib/index.js
@@ -1097,7 +1097,7 @@ exports.schnorr = {
 Point.BASE._setWindowSize(8);
 const crypto = {
     node: nodeCrypto,
-    web: typeof self === 'object' && 'crypto' in self ? self.crypto : undefined,
+    web: undefined,
 };
 const TAGS = {
     challenge: 'BIP0340/challenge',
