diff --git a/node_modules/@noble/ed25519/lib/esm/index.js b/node_modules/@noble/ed25519/lib/esm/index.js
index d763e8e..b5b2a8f 100644
--- a/node_modules/@noble/ed25519/lib/esm/index.js
+++ b/node_modules/@noble/ed25519/lib/esm/index.js
@@ -803,7 +803,7 @@ export const curve25519 = {
 };
 const crypto = {
     node: nodeCrypto,
-    web: typeof self === 'object' && 'crypto' in self ? self.crypto : undefined,
+    web: undefined
 };
 export const utils = {
     bytesToHex,
diff --git a/node_modules/@noble/ed25519/lib/index.js b/node_modules/@noble/ed25519/lib/index.js
index 50dd16d..805a3e8 100644
--- a/node_modules/@noble/ed25519/lib/index.js
+++ b/node_modules/@noble/ed25519/lib/index.js
@@ -813,7 +813,7 @@ exports.curve25519 = {
 };
 const crypto = {
     node: nodeCrypto,
-    web: typeof self === 'object' && 'crypto' in self ? self.crypto : undefined,
+    web: undefined
 };
 exports.utils = {
     bytesToHex,
