diff --git a/node_modules/eccrypto/index.js b/node_modules/eccrypto/index.js
index a63218b..0936a8d 100644
--- a/node_modules/eccrypto/index.js
+++ b/node_modules/eccrypto/index.js
@@ -12,18 +12,7 @@ var promise = typeof Promise === "undefined" ?
               require("es6-promise").Promise :
               Promise;
 var crypto = require("crypto");
-// try to use secp256k1, fallback to browser implementation
-try {
-  var secp256k1 = require("secp256k1");
-  var ecdh = require("./build/Release/ecdh");
-} catch (e) {
-  if (process.env.ECCRYPTO_NO_FALLBACK) {
-    throw e;
-  } else {
-    console.info('secp256k1 unavailable, reverting to browser version');
-    return (module.exports = require("./browser"));
-  }
-}
+var secp256k1 = require("secp256k1");

 function isScalar (x) {
   return Buffer.isBuffer(x) && x.length === 32;
