diff --git a/node_modules/webcrypto-core/build/webcrypto-core.js b/node_modules/webcrypto-core/build/webcrypto-core.js
index 8c1f27a..589727a 100644
--- a/node_modules/webcrypto-core/build/webcrypto-core.js
+++ b/node_modules/webcrypto-core/build/webcrypto-core.js
@@ -409,9 +409,9 @@ class AesGcmProvider extends AesProvider {
         if (algorithm.iv.byteLength < 1) {
             throw new OperationError("iv: Must have length more than 0 and less than 2^64 - 1");
         }
-        if (!("tagLength" in algorithm)) {
-            algorithm.tagLength = 128;
-        }
+
+        algorithm.tagLength ??= 128;
+        
         switch (algorithm.tagLength) {
             case 32:
             case 64:
